hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/wordcount/WordCount.java

```diff
@@ -111,15 +111,29 @@ public static void main(String[] args) throws Exception {
         Configuration conf = new Configuration();
         String[] otherArgs = new GenericOptionsParser(conf, args).getRemainingArgs();
         if (otherArgs.length < 5) {
-            System.err.println("Usage: wordcount <dummy_hdfs> <uri> <scope> <stream> <out>");
+            System.err.println("Usage: wordcount <dummy_hdfs> <uri> <scope> <stream> <out> <optional start positions> <optional end positions>");
```

Comment:
Maybe this is more specific: `<optional start stream cut>` / `<optional end stream cut>`

Reply:
accepted.